cmd/alias-import.go

```diff
@@ -0,0 +1,160 @@
+   {{ .Prompt }} {{ .HelpName }} exampleAlias/ ./credentials.json
+
+2. Import the credentials through standard input as 'exampleAlias'  to the config:
+   {{ .Prompt }} cat credentials.json | {{ .HelpName }} exampleAlias/
+
+`,
+}
+
+// checkAliasImportSyntax - verifies input arguments to 'alias import'.
+func checkAliasImportSyntax(ctx *cli.Context) {
+	args := ctx.Args()
+	argsNr := len(args)
+
+	if argsNr == 0 {
+		cli.ShowCommandHelpAndExit(ctx, ctx.Command.Name, 1)
+	}
+	if argsNr != 2 {
+		fatalIf(errInvalidArgument().Trace(ctx.Args().Tail()...),
+			"Incorrect number of arguments for alias Import command.")
+	}
+
+	alias := cleanAlias(args.Get(0))
+
+	if !isValidAlias(alias) {
+		fatalIf(errInvalidAlias(alias), "Invalid alias.")
+	}
+}
+
+func checkCredentialsSyntax(credentials aliasConfigV10) {
+	if !isValidHostURL(credentials.URL) {
+		fatalIf(errInvalidURL(credentials.URL), "Invalid URL.")
+	}
+
+	if !isValidAccessKey(credentials.AccessKey) {
+		fatalIf(errInvalidArgument().Trace(credentials.AccessKey),
+			"Invalid access key `"+credentials.AccessKey+"`.")
+	}
+
+	if !isValidSecretKey(credentials.SecretKey) {
+		fatalIf(errInvalidArgument().Trace(credentials.SecretKey),
+			"Invalid secret key `"+credentials.SecretKey+"`.")
+	}
+
+	if credentials.API != "" && !isValidAPI(credentials.API) { // Empty value set to default "S3v4".
+		fatalIf(errInvalidArgument().Trace(credentials.API),
+			"Unrecognized API signature. Valid options are `[S3v4, S3v2]`.")
+	}
+	if !isValidPath(credentials.Path) {
+		fatalIf(errInvalidArgument().Trace(credentials.Path),
+			"Unrecognized path value. Valid options are `[auto, on, off]`.")
+	}
+}
+
+// importAlias - set an alias config based on imported values.
+func importAlias(alias string, aliasCfgV10 aliasConfigV10) aliasMessage {
+	checkCredentialsSyntax(aliasCfgV10)
+
+	mcCfgV10, err := loadMcConfig()
+	fatalIf(err.Trace(globalMCConfigVersion), "Unable to load config `"+mustGetMcConfigPath()+"`.")
+
+	// Add new host.
+	mcCfgV10.Aliases[alias] = aliasCfgV10
+	fatalIf(saveMcConfig(mcCfgV10).Trace(alias), "Unable to import credentials to `"+mustGetMcConfigPath()+"`.")
+	return aliasMessage{
+		Alias:     alias,
+		URL:       mcCfgV10.Aliases[alias].URL,
+		AccessKey: mcCfgV10.Aliases[alias].AccessKey,
+		SecretKey: mcCfgV10.Aliases[alias].SecretKey,
+		API:       mcCfgV10.Aliases[alias].API,
+		Path:      mcCfgV10.Aliases[alias].Path,
+	}
+}
+
+func mainAliasImport(cli *cli.Context) error {
+	// console.SetColor("AliasMessage", color.New(color.FgGreen))
+	var (
+		args      = cli.Args()
+		alias     = cleanAlias(args.Get(0))
+		credsFile = args.Get((1))
+	)
+
+	checkAliasImportSyntax(cli)
+
+	var credentialsJSON aliasConfigV10
+
+	input, e := os.ReadFile(credsFile)
+
+	fatalIf(probe.NewError(e).Trace(args...), "Unable to parse credentials file")
+
+	json.Unmarshal([]byte(input), &credentialsJSON)
+
+	msg := importAlias(alias, credentialsJSON)
+
+	msg.op = "import"
+
+	printMsg(msg)
+
+	return nil
+}
```

Comment:
Okay, a slight change might be needed here to support reading os.Stdin optionally.

```
cat credentials.json | mc alias import alias/
```

So the code is perhaps like this - we do allow only alias as arg at that point  for example

```
if cli.NArg() == 1  {
     os.ReadFile(os.Stdin)
} else if cli.NArg() == 2 {
    os.ReadFile(cli.Args().Get(1))
}
```

This would end up supporting both `cat credentials.json | mc alias import alias/`  as well as `mc alias import alias/ ./credentials.json` 

You would have to relax the number of args in the Syntax(ctx) function appropriately.